setup.py

```diff
@@ -99,5 +92,7 @@ def build_extension(self, ext, cmake):
               'neurolucida'
               'neuromorphology'),
     zip_safe=False,
-    classifiers=[]
+    classifiers=[],
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
```

Comment:
hmm, I find the issue list for setuptools_scm a bit concerning - quite a lot of old issues, and it seems if it's not being used in the *exact* manor, it's hard to work around (ie: issue 304)

Reply:
On the other hand, the repo is hosted in `pypa` and seems to be alive so far.
Also, I don't know if we ever need anything exotic from it. ^^

What are our alternatives for pairing version as seen by C++ / Python / Git tags?